app/assets/stylesheets/sufia/_collections.scss

```diff
@@ -79,3 +79,7 @@ form.per_page {
     padding-left: 0;
   }
 }
+
+.collection_form_visibility .form-group {
+  padding-left: 25px;
```

Comment:
I think the same style is applied in CC here: https://github.com/projecthydra-labs/curation_concerns/blob/71f0b9d8b210a7833eefdf333bfeb8859e255c56/app/assets/stylesheets/curation_concerns/modules/forms.scss#L43
 Should we use em's instead of px here?